.travis.yml

```diff
@@ -36,3 +24,84 @@ cache:
   - "$HOME/.coursier/cache"
   - "$HOME/.ivy2/cache"
   - "$HOME/.sbt/boot"
+
+jobs:
+  include:
+    - &tests
```

Comment:
I don't understand this syntax or what invokes it.  Is there a reference so I can learn?